synapse/storage/databases/main/schema/delta/58/11fallback.sql

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TABLE IF NOT EXISTS e2e_fallback_keys_json (
```

Comment:
Seems like this essentially corresponds to `e2e_one_time_keys_json`. That has a `ts_added_ms` -- curious if something similar is useful here or not?

Reply:
Yeah.  I don't think we even use `ts_added_ms` in `e2e_one_time_keys_json` (it might be useful there for making sure that old keys get claimed before new ones, but we don't even use it for that.), so I didn't bother adding it.

Reply:
Makes sense, just wanted to make sure I wasn't missing something.